exercises/diffie-hellman/tests/diffie-hellman.rs

```diff
@@ -67,6 +67,63 @@ fn test_secret_key_correct_big_numbers() {
     assert_eq!(secret, expected);
 }
 
+// two biggest 64bit primes
+#[allow(dead_code)]
```

Comment:
```suggestion
#[cfg(feature = "big-primes")]
```

These consts are only dead code when the appropriate tests are ignored during compilation, so may as well apply the same feature gate to the consts instead of compiling them unconditionally.

Reply:
I wondered if there was a better solution